packages/playground/website/playwright/playground-fixtures.ts

```diff
@@ -0,0 +1,50 @@
+import { test as base, Page, expect, FrameLocator } from '@playwright/test';
+
+type WordPressFixtures = {
+	wordpressPage: FrameLocator;
+	page: Page;
+};
+
+export const test = base.extend<WordPressFixtures>({
+	wordpressPage: async ({ page }, use) => {
+		const wpPage = page
+			.frameLocator('#playground-viewport')
+			.frameLocator('#wp');
+		await use(wpPage);
+	},
+	page: async ({ page }, use) => {
+		// Define a function to wait for nested iframes
+		async function waitForNestedIframes(page: Page) {
+			const timeout = 120000;
+			const outerSelector = '#playground-viewport';
+			await page.waitForSelector(outerSelector, {
+				timeout,
+			});
+			const outerFrame = page.frameLocator(outerSelector);
+			await expect(outerFrame.locator('body')).not.toBeEmpty({
+				timeout,
+			});
+
+			const innerSelector = '#wp';
+			await outerFrame.locator(innerSelector).waitFor({
+				timeout,
+			});
+			const innerFrame = outerFrame.frameLocator(innerSelector);
+			await expect(innerFrame.locator('body')).not.toBeEmpty({
+				timeout,
+			});
+			return innerFrame;
+		}
+
+		// Extend the original page.goto method
+		const originalGoto = page.goto.bind(page);
+		page.goto = async (url: string, options?: any) => {
+			const response = await originalGoto(url, options);
+			await waitForNestedIframes(page);
+			return response;
+		};
+		await use(page);
+	},
```

Comment:
I added this to ensure Playground is fully loaded after a `goto` without it `enableMultisite` fails.
I assume it fails because it does requests internally.

I cleaned up the `waitForNestedIframes` based on your feedback.

I will take another look at removing this after https://github.com/WordPress/wordpress-playground/pull/1789 is merge because it has similar issues with the multisite step.

> In case the navigation times out, then the default [timeout value for the navigation](https://playwright.dev/docs/test-timeouts#set-action-and-navigation-timeouts-in-the-config) can be increased.

Thanks! I made this change now.